cita-network/src/network.rs

```diff
@@ -291,3 +292,21 @@ impl RemoteMessage {
         }
     }
 }
+
+pub fn send_message(nodes_mgr_client: &NodesManagerClient, key: String, msg: ProtoMessage) {
+    let operate = msg.get_operate();
+
+    match operate {
+        OperateType::Broadcast => {
+            nodes_mgr_client.broadcast(BroadcastReq::new(key, msg));
+        }
+        OperateType::Single => {
+            let dst = msg.get_origin();
+            nodes_mgr_client.send_message(SingleTxReq::new(dst as usize, key, msg));
+        }
+        OperateType::Subtract => {
```

Comment:
At least, broadcast to all.

Reply:
That sounds more reasonable.

Reply:
What's subtract broadcast?

Reply:
> What's subtract broadcast?

- `Broadcast()` means to all.
- `Single()` means to the specify node.
- `Subtract()` means to all except the specify node.

Reply:
Broadcast to all nodes except the specified one.

Reply:
Got it.